pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -44,24 +44,63 @@ Quick link:
+
+While the WCAG2ICT Group Note provides guidance on applying WCAG in non-web contexts, at this time it may not provide sufficient clarity for some success criteria on how they can be applied to electronic documents whose source is driven by non-web markup technologies when they are delivered via a website. Likewise, there may not be sufficient clarity for mobile applications.  
 
-*Scope*: WCAG2ICT does not include techniques for how to meet WCAG success criteria in specific technologies. More information is in the [Excluded from Scope section](https://www.w3.org/TR/wcag2ict-22/#excluded-from-scope).
 
 ### Versions
-* [Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) was published 8 October 2024 and covers WCAG 2.0, 2.1, and 2.2
+* [Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) was published 15 November 2024 and covers WCAG 2.0, 2.1, and 2.2
 * older version: [Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-20/) was published 5 September 2013 and covers WCAG 2.0
 * The URI [https://www.w3.org/TR/wcag2ict/](https://www.w3.org/TR/wcag2ict/) points to the latest completed version
 
 ## Who WCAG2ICT is for {#for}
 
-WCAG2ICT is for ICT managers, ICT developers, policy makers, and others wanting to understand how WCAG 2 can be applied to non-web documents and software.
+The WCAG2ICT Group Note is for standards developers, ICT managers, ICT developers, policy makers, and any others wanting to understand how WCAG 2 can be applied to non-web documents and software.
+* Accessibility [standards organizations](https://en.wikipedia.org/wiki/Standards_organization) - to provide non-web language in the success criteria and definitions and to provide interpretation notes to help the organization make any potential adjustments or notes needed for specific technologies, if needed.
+* ICT managers  - who are familiar with WCAG to quickly get an overview of using WCAG as an accessibility benchmark for non-web ICT.
+* Technology developers - to provide the language substitutions for non-web language in WCAG success criteria where existing accessibility standards don’t contain an interpretation.
+* Policy makers (which includes legislators, regulators) - to inform policy makers how the WCAG2ICT note is being used to influence standards requirements where WCAG is being applied to non-web ICT.
```

Comment:
```suggestion
* Policy makers (including regulators when making regulations) - to inform policy makers how the WCAG2ICT note is being used to influence standards requirements where WCAG is being applied to non-web ICT.
```

Reply:
Suggesting this instead of @daniel-montalvo suggested edits per the TF meeting on 17 April.